doc/spectrographs/ldt_deveny.rst

```diff
@@ -1,18 +1,1799 @@
+-----------------
+
+PypeIt is designed (and currently only able) to do end-to-end reductions,
+resulting in a fully processed 2D spectral image and extracted 1D spectra (if
+any objects were found) from each science frames. Once you have completed the
+setup steps above, you are just about ready to run the pipeline.
+
+The script to run a reduction is :ref:`run-pypeit`.  See that documentation
+page for all relevant script options and workflows.
+
+.. caution::
+
+   When you upgrade PypeIt versions, changes to the underlying data models
+   (which are largely not backwards compatible) may cause errors if you try to
+   use calibration files processed with an earlier version. The safe move is to
+   completely reprocess all data currently being used when PypeIt is upgraded,
+   including deleting and recreating all processed calibrations.  Your
+   currently installed version of PypeIt may be checked using
+   ``pypeit_version``, and the version used to create any output file is listed
+   in the FITS header with the keyword ``VERSPYP``.
+
+
+.. _deveny_outputs:
+
+Primary Output Files and Post-Processing Scripts
+================================================
+
+.. _deveny_calibrations:
+
+Examine the Calibration Files
+-----------------------------
+
+As PypeIt begins churning through your reduction, it will create and write to
+disk calibration frames in the ``Calibrations/`` subfolder of the, *e.g.*,
+``ldt_deveny_A/`` directory (see :ref:`deveny_filestructure`). Additional
+Quality Assurance files will be written to the ``QA/`` subfolder for some
+types of Calibration frames. It is important to take the time to inspect these
+calibration outputs as they are generated.
+
+.. tip::
+   
+   To process just the calibrations without trying to process the ``science``
+   data, use the command::
+
+      run_pypeit -c ldt_deveny_<setup>.pypeit
+
+The naming convention for :ref:`calibrations` frames is a bit cumbersome, but
+follows a regular pattern.  Here is a brief listing of the Calibration frames
+produced (in the order in which they are created):
+
+-  :ref:`bias` -- Processed combined bias frame used to remove
+   fixed-pattern noise from all other images.
+
+-  :ref:`edges` -- Collection of images and FITS binary tables
+   describing the slit traces. While this file is primarily of interest
+   for multislit or echelle spectrographs (DeVeny has but one slit and
+   no cross-disperser, after all), it is instructive to quickly peek at
+   this file to ensure the code correctly identified the slit (and not
+   some artifact at the edge of the CCD):
+
+   ::
+
+      $ pypeit_chk_edges Calibrations/Edges_A_0_DET01.fits.gz
+
+   This command will launch a :ref:`GUI viewer <pypeit_chk_edges>` to display
+   the combined trace image along with a sobel-filtered version used to
+   identify illumination discontinuities in the spatial direction (see figure
+   below). For DeVeny data, it should identify a single, wide longslit with a
```

Comment:
At the risk of harping on this, I associate "wide" and "narrow" with dimensions along the dispersion direct, and "short" and "long" with spatial extent.  So my original comment on this was mostly advocating that we try to minimize potential confusion by keeping that adjective association. FWIW, I think the sentence is more clear if you remove the word "wide."